EGO Forum Enhancement.user.js

```diff
@@ -238,10 +264,15 @@ function handleForumsList() {
                 }
                 var userHref = thread.querySelector('.structItem-cell--main > .structItem-minor > .structItem-parts > li > a');
                 var threadTitle = thread.querySelector('.structItem-cell--main > .structItem-title > a');
+
+                var titleHtml = threadTitle.outerHTML;
+                if(threadTitle.classList.contains('labelLink')) {
```

Comment:
Inconsistent spacing

```suggestion
                if (threadTitle.classList.contains('labelLink')) {
```